packages/network-controller/src/NetworkController.ts

```diff
@@ -293,10 +286,6 @@ export class NetworkController extends BaseControllerV2<
     ticker?: string,
     nickname?: string,
   ) {
-    this.update((state) => {
-      state.isCustomNetwork = this.#getIsCustomNetwork(chainId);
```

Comment:
Should we remove `#getIsCustomNetwork` as well?

Reply:
I'd prefer to keep it around for sentimental reasons. As a memento, you know.

Reply:
Just kidding, good catch, removed in https://github.com/MetaMask/core/pull/1199/commits/ae8d073a10e72cdd64c9953315616501851ae34e